packages/fxa-auth-server/lib/payments/stripe.js

```diff
@@ -738,6 +738,11 @@ class StripeHelper {
       };
 
       const planName = `${productName} ${intervalDict[sub.plan.interval]}`;
+      let latestInvoice = sub.latest_invoice;
+
+      if (latestInvoice && typeof latestInvoice !== 'string') {
+        latestInvoice = latestInvoice.number;
```

Comment:
We probably would want to be using `latestInvoice.id` when we have an invoice object. While there is an `latestInvoice.number`, it is not the same thing that we would see if `latestInvoice` is not expanded. So, for consistency, there are two options:
1. use `id` field, when not expanded
2. if the invoice is not expanded, fetch it, and always use `number`

Reply:
👍 TY Jackie, I'm going to link your comment in #4311.